docs/connection-options.md

```diff
@@ -82,7 +82,7 @@ This option is useful during debug and development.
  Note that for MongoDB database it does not create schema, because MongoDB is schemaless.
  Instead, it syncs just by creating indices.
 
-* `migrationsRun` - Indicates if migrations should be auto run on every application launch.
+* `autoMigrationsRun` - Indicates if migrations should be auto run on every application launch.
```

Comment:
what typeorm version are you using? In latest version it does not call `autoMigrationsRun` anymore. Now its called `migrationsRun`

Reply:
I'm using version: 
`    "typeorm": "0.1.0-alpha.31"`

I did an npm-check and it did not show any updates.
But I see it's aplha.50 now.

Sorry!

Reply:
np, I'll close this pull request then. Hope to see your contribution in new PRs!